support/package_modsecurity

```diff
@@ -51,36 +50,34 @@ if [[ "$STACK" =~ .*minimal$ ]] ; then
   deps_stack=$(echo $STACK | sed -e 's/-minimal//')
 fi
 
-declare -A deps_versions
-
 status "Packaging libgeoip ${geoip_version[$deps_stack]}"
 
-curl --remote-name --location "${pkgs_url}/g/geoip/libgeoip-dev_${geoip_version[$deps_stack]}_amd64.deb"
-curl --remote-name --location "${pkgs_url}/g/geoip/libgeoip1_${geoip_version[$deps_stack]}_amd64.deb"
+curl --remote-name --location "${pkgs_url}/main/g/geoip/libgeoip-dev_${geoip_version[$deps_stack]}_amd64.deb"
+curl --remote-name --location "${pkgs_url}/main/g/geoip/libgeoip1_${geoip_version[$deps_stack]}_amd64.deb"
 extract_deb "libgeoip-dev_${geoip_version[$deps_stack]}_amd64.deb" "${install_dir}"
 extract_deb "libgeoip1_${geoip_version[$deps_stack]}_amd64.deb" "${install_dir}"
 
 status "Packaging libyajl ${yajl_version[$deps_stack]}"
-curl --remote-name --location --silent "${pkgs_url}/y/yajl/libyajl-dev_${yajl_version[$deps_stack]}_amd64.deb"
-curl --remote-name --location --silent "${pkgs_url}/y/yajl/libyajl2_${yajl_version[$deps_stack]}_amd64.deb"
+curl --remote-name --location --silent "${pkgs_url}/main/y/yajl/libyajl-dev_${yajl_version[$deps_stack]}_amd64.deb"
+curl --remote-name --location --silent "${pkgs_url}/main/y/yajl/libyajl2_${yajl_version[$deps_stack]}_amd64.deb"
 extract_deb "libyajl-dev_${yajl_version[$deps_stack]}_amd64.deb" "${install_dir}"
 extract_deb "libyajl2_${yajl_version[$deps_stack]}_amd64.deb" "${install_dir}"
 
-status "Packaging liblmdb ${lmdb_repo[$deps_stack]}"
-curl --remote-name --location --silent "https://mirrors.kernel.org/ubuntu/pool/${lmdb_repo[$deps_stack]}/l/lmdb/liblmdb-dev_${lmdb_version[$deps_stack]}_amd64.deb"
```

Comment:
I tested and it works just fine to use `mirrors.edge.kernel.org` everywhere. I don't know the reason why we had two different mirrors URL used